notebooks/adalflow_dataclasses.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<p>Try to use &lt;USER&gt;&lt;/USER&gt; to replace User: This fits more on how LLMs are using special tokens</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SylphAI-Inc/AdalFlow/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
i have addressed the issue